gslib/tests/test_signurl.py

```diff
@@ -278,7 +279,10 @@ def testSignurlPutContentype(self):
 
   def testSignurlGet(self):
     """Tests the _GenSignedUrl function with a GET method."""
-    expected = sigs.TEST_SIGN_URL_GET
+    if PY2:
```

Comment:
This can be removed since we're using the same set of hashes now per earlier PR feedback.